docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,370 @@
-# Overview
+# Machine learning with TensorFlow and Mbed OS
+<!--https://github.com/COTASPAR/K66F/blob/master/README.md-->
+As an IoT developer, you might think of machine learning as a server-side technology: sensors on your device capture data and send it to the cloud, where Machine Learning (ML) models on hefty machines make sense of it. A network connection is obligatory, and you are going to expect some latency, not to mention hosting costs.
 
-As an IoT developer, you might think of machine learning as a server-side technology. In the traditional view, sensors on your device capture data and send it to the cloud, where Machine Learning (ML) models on hefty machines make sense of it. A network connection is obligatory, and you are going to expect some latency, not to mention hosting costs.
+But with the launch of [TensorFlow Lite for Microcontrollers](https://www.tensorflow.org/lite/microcontrollers), you can run machine learning inference on extremely low-powered devices, like the Cortex-M microcontroller series. You can deploy your ML models to the IoT devices themselves, bring ML closer to your sensors, remove your reliance on a network connection, and skip the round trip to the server for lower latency.
```

Comment:
The question below on the [VMs](https://github.com/ARMmbed/mbed-os-5-docs/pull/1369#discussion_r512669546) doesn't let me respond so I'll say here: Yes, I  agree. Should we just say tested on Ubuntu 20.04? rather than walk users through how to get Ubuntu on their machine.